resources/views/components/translatable-input.blade.php

```diff
@@ -7,7 +7,10 @@
                 @else
                     <x-hearth-label :for="$name . '_' . $language" :value="$label . ' (' . get_language_exonym($language) . ')'" />
                 @endif
-                <x-interpretation class="interpretation--start" name="{{ __($label, [], 'en') }}" />
+                @if ($interpretationName)
+                    <x-interpretation class="interpretation--start" name="{{ __($interpretationName, [], 'en') }}"
+                        namespace={{ $interpretationNameSpace }} />
+                @endif
```

Comment:
Missing the "" marks around the `namespace` value.

Reply:
Defaulting `$interpretationName` and `$interpretationNameSpace` to `null` can used `@isset` instead of `@if`. You can also used this internally to set the `namespace` value only when `$interpretationNameSpace` or allow it to pass down `null` but would need to make sure it doesn't pass the string "null".